programs/ssl/ssl_pthread_server.c

```diff
@@ -331,6 +334,14 @@ int main( void )
     mbedtls_ssl_cache_context cache;
 #endif
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet elsewhere are displaying it without the negative. We should be self-consistent within the same program.